src/PKSim.Core/Model/SimulationFactory.cs

```diff
@@ -109,6 +109,7 @@ public Simulation CreateFrom(ISimulationSubject simulationSubject, IReadOnlyList
 
          _simulationBuildingBlockUpdater.UpdateUsedBuildingBlockInSimulationFromTemplate(simulation, simulationSubject, PKSimBuildingBlockType.SimulationSubject);
          _simulationBuildingBlockUpdater.UpdateMultipleUsedBuildingBlockInSimulationFromTemplate(simulation, compounds, PKSimBuildingBlockType.Compound);
+         _simulationBuildingBlockUpdater.UpdateMultipleUsedBuildingBlockInSimulationFromTemplate(simulation, simulationSubject.AllExpressionProfiles(), PKSimBuildingBlockType.ExpressionProfile);
```

Comment:
When we create a simulation, we need to all also all expression profiles as USED building blocks so that they are serialized as well into the project. It won't be available when loading v11 but so is life. We could try to add them if the simulation is green etc.. to be disucssed

Reply:
let's create a new Issue for this